domain/domain_sysvars.go

```diff
@@ -76,7 +76,10 @@ func (do *Domain) setGlobalResourceControl(enable bool) {
 	} else {
 		variable.DisableGlobalResourceControlFunc()
 	}
-	logutil.BgLogger().Info("set resource control", zap.Bool("enable", enable))
+
+	if enable != variable.EnableResourceControl.Load() {
```

Comment:
This is false negative, because `variable.EnableResourceControl` is set to `enable` before this function is called, so this log will never be printed.

Reply:
fixed.